code/go/0chain.net/blobber/config.go

```diff
@@ -40,16 +51,28 @@ func setupConfig(configDir string, deploymentMode int) {
 	config.Configuration.ChallengeResolveNumWorkers = viper.GetInt("challenge_response.num_workers")
 	config.Configuration.ChallengeMaxRetires = viper.GetInt("challenge_response.max_retries")
 
+	config.Configuration.AutomaticUpdate = viper.GetBool("disk_update.automatic_update")
+	blobberUpdateIntrv := viper.GetInt("disk_update.blobber_update_interval")
```

Comment:
From user side, it would be easy to put integer value in minute or second. 
One second would be equivalent to `10^9` duration value.

I think it is better to keep it as it is.

Reply:
User only need to set `1m` or `1s` for 1 minute or 1 second, the viper.GetDuration() will parse the string to time.Duration for us.